### ISSUE TITLE
java.lang.NullPointerException

### ISSUE THREAD

**Issue by blaher** [awslabs/aws-codepipeline-plugin-for-jenkins#31]
I'm currently using 0.34 on Jenkins 2.110. Whenever I try to select AWS CodePipeline under Source Code Management and save it, I get a stack trace error. I previously had projects working prior to upgrading Jenkins, but now those projects will no longer save. Here is the error:

`
java.lang.NullPointerException
	at com.amazonaws.codepipeline.jenkinsplugin.AWSCodePipelineSCM.<init>(AWSCodePipelineSCM.java:137)
	at com.amazonaws.codepipeline.jenkinsplugin.AWSCodePipelineSCM$DescriptorImpl.newInstance(AWSCodePipelineSCM.java:378)
	at com.amazonaws.codepipeline.jenkinsplugin.AWSCodePipelineSCM$DescriptorImpl.newInstance(AWSCodePipelineSCM.java:346)
	at hudson.DescriptorExtensionList.newInstanceFromRadioList(DescriptorExtensionList.java:142)
	at hudson.scm.SCMS.parseSCM(SCMS.java:59)
	at hudson.model.AbstractProject.submit(AbstractProject.java:1802)
	at hudson.model.Project.submit(Project.java:225)
	at hudson.model.Job.doConfigSubmit(Job.java:1350)
	at hudson.model.AbstractProject.doConfigSubmit(AbstractProject.java:772)
	at java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:627)
	at org.kohsuke.stapler.Function$MethodFunction.invoke(Function.java:343)
	at org.kohsuke.stapler.interceptor.RequirePOST$Processor.invoke(RequirePOST.java:77)
	at org.kohsuke.stapler.PreInvokeInterceptedFunction.invoke(PreInvokeInterceptedFunction.java:26)
	at org.kohsuke.stapler.Function.bindAndInvoke(Function.java:184)
	at org.kohsuke.stapler.Function.bindAndInvokeAndServeResponse(Function.java:117)
	at org.kohsuke.stapler.MetaClass$1.doDispatch(MetaClass.java:129)
	at org.kohsuke.stapler.NameBasedDispatcher.dispatch(NameBasedDispatcher.java:58)
	at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:715)
Caused: javax.servlet.ServletException
	at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:765)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:845)
	at org.kohsuke.stapler.MetaClass$5.doDispatch(MetaClass.java:248)
	at org.kohsuke.stapler.NameBasedDispatcher.dispatch(NameBasedDispatcher.java:58)
	at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:715)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:845)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:649)
	at org.kohsuke.stapler.Stapler.service(Stapler.java:238)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:841)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1650)
	at hudson.util.PluginServletFilter$1.doFilter(PluginServletFilter.java:154)
	at hudson.util.PluginServletFilter.doFilter(PluginServletFilter.java:157)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at hudson.security.csrf.CrumbFilter.doFilter(CrumbFilter.java:64)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:84)
	at hudson.security.UnwrapSecurityExceptionFilter.doFilter(UnwrapSecurityExceptionFilter.java:51)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at jenkins.security.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:117)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at org.acegisecurity.providers.anonymous.AnonymousProcessingFilter.doFilter(AnonymousProcessingFilter.java:125)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at org.acegisecurity.ui.rememberme.RememberMeProcessingFilter.doFilter(RememberMeProcessingFilter.java:142)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at org.acegisecurity.ui.AbstractProcessingFilter.doFilter(AbstractProcessingFilter.java:271)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at jenkins.security.BasicHeaderProcessor.doFilter(BasicHeaderProcessor.java:93)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at org.acegisecurity.context.HttpSessionContextIntegrationFilter.doFilter(HttpSessionContextIntegrationFilter.java:249)
	at hudson.security.HttpSessionContextIntegrationFilter2.doFilter(HttpSessionContextIntegrationFilter2.java:67)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at hudson.security.ChainedServletFilter.doFilter(ChainedServletFilter.java:90)
	at hudson.security.HudsonFilter.doFilter(HudsonFilter.java:171)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at org.kohsuke.stapler.compression.CompressionFilter.doFilter(CompressionFilter.java:49)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at hudson.util.CharacterEncodingFilter.doFilter(CharacterEncodingFilter.java:82)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at org.kohsuke.stapler.DiagnosticThreadNameFilter.doFilter(DiagnosticThreadNameFilter.java:30)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:533)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:524)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:190)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1595)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1253)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:168)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:473)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1564)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:166)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1155)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:564)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:317)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:110)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124)
	at org.eclipse.jetty.util.thread.Invocable.invokePreferred(Invocable.java:128)
	at org.eclipse.jetty.util.thread.Invocable$InvocableExecutor.invoke(Invocable.java:222)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:294)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:199)
	at winstone.BoundedExecutorService$1.run(BoundedExecutorService.java:77)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
`

**Comment by galmeida** [comment]
Can you please check if your project name is null or just blank in either Jenkins or CodePipeline jenkins action?

**Comment by blaher** [comment]
I can confirm we have project names set for both.

**Comment by itspoma** [comment]
confirming the same issue.

![](https://www.dropbox.com/s/b15fnnwc95nvzlb/Screenshot%202018-03-12%2011.55.25.png?raw=1)

```
java.lang.NullPointerException
	at com.amazonaws.codepipeline.jenkinsplugin.AWSCodePipelineSCM.<init>(AWSCodePipelineSCM.java:137)
	at com.amazonaws.codepipeline.jenkinsplugin.AWSCodePipelineSCM$DescriptorImpl.newInstance(AWSCodePipelineSCM.java:378)
	at com.amazonaws.codepipeline.jenkinsplugin.AWSCodePipelineSCM$DescriptorImpl.newInstance(AWSCodePipelineSCM.java:346)
	at hudson.DescriptorExtensionList.newInstanceFromRadioList(DescriptorExtensionList.java:142)
	at hudson.scm.SCMS.parseSCM(SCMS.java:59)
	at hudson.model.AbstractProject.submit(AbstractProject.java:1802)
	at hudson.model.Project.submit(Project.java:225)
	at hudson.model.Job.doConfigSubmit(Job.java:1350)
	at hudson.model.AbstractProject.doConfigSubmit(AbstractProject.java:772)
	at java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:627)
	at org.kohsuke.stapler.Function$MethodFunction.invoke(Function.java:343)
	at org.kohsuke.stapler.interceptor.RequirePOST$Processor.invoke(RequirePOST.java:77)
	at org.kohsuke.stapler.PreInvokeInterceptedFunction.invoke(PreInvokeInterceptedFunction.java:26)
	at org.kohsuke.stapler.Function.bindAndInvoke(Function.java:184)
	at org.kohsuke.stapler.Function.bindAndInvokeAndServeResponse(Function.java:117)
	at org.kohsuke.stapler.MetaClass$1.doDispatch(MetaClass.java:129)
	at org.kohsuke.stapler.NameBasedDispatcher.dispatch(NameBasedDispatcher.java:58)
	at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:715)
Caused: javax.servlet.ServletException
	at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:765)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:845)
	at org.kohsuke.stapler.MetaClass$5.doDispatch(MetaClass.java:248)
	at org.kohsuke.stapler.NameBasedDispatcher.dispatch(NameBasedDispatcher.java:58)
	at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:715)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:845)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:649)
	at org.kohsuke.stapler.Stapler.service(Stapler.java:238)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:841)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1650)
	at hudson.util.PluginServletFilter$1.doFilter(PluginServletFilter.java:154)
	at org.jenkinsci.plugins.ssegateway.Endpoint$SSEListenChannelFilter.doFilter(Endpoint.java:225)
	at hudson.util.PluginServletFilter$1.doFilter(PluginServletFilter.java:151)
	at io.jenkins.blueocean.ResourceCacheControl.doFilter(ResourceCacheControl.java:134)
	at hudson.util.PluginServletFilter$1.doFilter(PluginServletFilter.java:151)
	at io.jenkins.blueocean.auth.jwt.impl.JwtAuthenticationFilter.doFilter(JwtAuthenticationFilter.java:61)
	at hudson.util.PluginServletFilter$1.doFilter(PluginServletFilter.java:151)
	at hudson.util.PluginServletFilter.doFilter(PluginServletFilter.java:157)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at hudson.security.csrf.CrumbFilter.doFilter(CrumbFilter.java:99)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:84)
	at hudson.security.UnwrapSecurityExceptionFilter.doFilter(UnwrapSecurityExceptionFilter.java:51)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at jenkins.security.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:117)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at org.acegisecurity.providers.anonymous.AnonymousProcessingFilter.doFilter(AnonymousProcessingFilter.java:125)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at org.acegisecurity.ui.rememberme.RememberMeProcessingFilter.doFilter(RememberMeProcessingFilter.java:142)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at org.acegisecurity.ui.AbstractProcessingFilter.doFilter(AbstractProcessingFilter.java:271)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at jenkins.security.BasicHeaderProcessor.doFilter(BasicHeaderProcessor.java:93)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at org.acegisecurity.context.HttpSessionContextIntegrationFilter.doFilter(HttpSessionContextIntegrationFilter.java:249)
	at hudson.security.HttpSessionContextIntegrationFilter2.doFilter(HttpSessionContextIntegrationFilter2.java:67)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:87)
	at hudson.security.ChainedServletFilter.doFilter(ChainedServletFilter.java:90)
	at hudson.security.HudsonFilter.doFilter(HudsonFilter.java:171)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at org.kohsuke.stapler.compression.CompressionFilter.doFilter(CompressionFilter.java:49)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at hudson.util.CharacterEncodingFilter.doFilter(CharacterEncodingFilter.java:82)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at org.kohsuke.stapler.DiagnosticThreadNameFilter.doFilter(DiagnosticThreadNameFilter.java:30)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:533)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:524)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:190)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1595)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1253)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:168)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:473)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1564)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:166)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1155)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:564)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:317)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:110)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124)
	at org.eclipse.jetty.util.thread.Invocable.invokePreferred(Invocable.java:128)
	at org.eclipse.jetty.util.thread.Invocable$InvocableExecutor.invoke(Invocable.java:222)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:294)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:199)
	at winstone.BoundedExecutorService$1.run(BoundedExecutorService.java:77)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

@galmeida can you please assist?

**Comment by galmeida** [comment]
thanks for reporting @itspoma, would you mind telling me what jenkins version are you running? on what operating system?

**Comment by itspoma** [comment]
@galmeida sure! According to `/systemInfo`:

```
Jenkins ver. 2.110
java.class.version | 52.
java.endorsed.dirs | /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/endorsed
java.ext.dirs | /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext:/usr/java/packages/lib/ext
java.home | /usr/lib/jvm/java-8-openjdk-amd64/jre
java.runtime.version | 1.8.0_151-8u151-b12-0ubuntu0.16.04.2-b12
java.version | 1.8.0_151
os.arch | amd64
os.name | Linux
os.version | 4.4.0-1049-aws
```

OS:
```
$ cat /etc/*release

DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
VERSION="16.04.3 LTS (Xenial Xerus)"
```

Here's all list of plugins:
```
Name  ↓ | Version | Enabled
-- | -- | --
ace-editor | 1.1 | true
amazon-ecr | 1.6 | true
ant | 1.8 | true
antisamy-markup-formatter | 1.5 | true
apache-httpcomponents-client-4-api | 4.5.3-2.1 | true
authentication-tokens | 1.3 | true
aws-codepipeline | 0.34 | true
aws-credentials | 1.23 | true
aws-java-sdk | 1.11.264 | true
blueocean | 1.4.2 | true
blueocean-autofavorite | 1.2.2 | true
blueocean-bitbucket-pipeline | 1.4.2 | true
blueocean-commons | 1.4.2 | true
blueocean-config | 1.4.2 | true
blueocean-core-js | 1.4.2 | true
blueocean-dashboard | 1.4.2 | true
blueocean-display-url | 2.2.0 | true
blueocean-events | 1.4.2 | true
blueocean-git-pipeline | 1.4.2 | true
blueocean-github-pipeline | 1.4.2 | true
blueocean-i18n | 1.4.2 | true
blueocean-jira | 1.4.2 | true
blueocean-jwt | 1.4.2 | true
blueocean-personalization | 1.4.2 | true
blueocean-pipeline-api-impl | 1.4.2 | true
blueocean-pipeline-editor | 1.4.2 | true
blueocean-pipeline-scm-api | 1.4.2 | true
blueocean-rest | 1.4.2 | true
blueocean-rest-impl | 1.4.2 | true
blueocean-web | 1.4.2 | true
bouncycastle-api | 2.16.2 | true
branch-api | 2.0.18 | true
build-timeout | 1.19 | true
cloudbees-bitbucket-branch-source | 2.2.10 | true
cloudbees-folder | 6.3 | true
command-launcher | 1.2 | true
config-file-provider | 2.17 | true
credentials | 2.1.16 | true
credentials-binding | 1.15 | true
display-url-api | 2.2.0 | true
docker-commons | 1.11 | true
docker-workflow | 1.15.1 | true
durable-task | 1.21 | true
email-ext | 2.61 | true
extended-choice-parameter | 0.76 | true
favorite | 2.3.1 | true
git | 3.8.0 | true
git-client | 2.7.1 | true
git-server | 1.7 | true
github | 1.29.0 | true
github-api | 1.90 | true
github-branch-source | 2.3.2 | true
gradle | 1.28 | true
handlebars | 1.1.1 | true
handy-uri-templates-2-api | 2.1.6-1.0 | true
htmlpublisher | 1.14 | true
jackson2-api | 2.8.11.1 | true
jenkins-design-language | 1.4.2 | true
jira | 2.5 | true
jquery | 1.12.4-0 | true
jquery-detached | 1.2.1 | true
jsch | 0.1.54.2 | true
junit | 1.24 | true
mailer | 1.20 | true
matrix-auth | 2.2 | true
matrix-project | 1.12 | true
mercurial | 2.3 | true
momentjs | 1.1.1 | true
nodejs | 1.2.5 | true
pipeline-build-step | 2.7 | true
pipeline-graph-analysis | 1.6 | true
pipeline-input-step | 2.8 | true
pipeline-milestone-step | 1.3.1 | true
pipeline-model-api | 1.2.7 | true
pipeline-model-declarative-agent | 1.1.1 | true
pipeline-model-definition | 1.2.7 | true
pipeline-model-extensions | 1.2.7 | true
pipeline-rest-api | 2.9 | true
pipeline-stage-step | 2.3 | true
pipeline-stage-tags-metadata | 1.2.7 | true
pipeline-stage-view | 2.9 | true
plain-credentials | 1.4 | true
pubsub-light | 1.12 | true
resource-disposer | 0.8 | true
scm-api | 2.2.6 | true
script-security | 1.41 | true
slack | 2.3 | true
sse-gateway | 1.15 | true
ssh-credentials | 1.13 | true
ssh-slaves | 1.26 | true
structs | 1.14 | true
timestamper | 1.8.9 | true
token-macro | 2.3 | true
variant | 1.1 | true
windows-slaves | 1.3.1 | true
workflow-aggregator | 2.5 | true
workflow-api | 2.26 | true
workflow-basic-steps | 2.6 | true
workflow-cps | 2.45 | true
workflow-cps-global-lib | 2.9 | true
workflow-durable-task-step | 2.19 | true
workflow-job | 2.17 | true
workflow-multibranch | 2.17 | true
workflow-scm-step | 2.6 | true
workflow-step-api | 2.14 | true
workflow-support | 2.18 | true
ws-cleanup | 0.34 | true
```

**Comment by galmeida** [comment]
Thanks, I'll update this thread soon

**Comment by galmeida** [comment]
The issue appears to be caused by incompatibility between AWS CodePipeline plugin and recent weekly jenkins releases.

I've already fixed it but still need to run some more tests before posting a new release. In the meanwhile if you are interested in trying my pre-release version, let me know how to get in touch privately and I will send you the instructions.

Thanks for reporting the issue.

**Comment by itspoma** [comment]
@galmeida let me know to itspoma@gmail.com, I'll be happy to test pre-release

**Comment by galmeida** [comment]
tks @itspoma, check your email, look for the subject "AWS CodePipeline Jenkins plugin"

**Comment by galmeida** [comment]
we just released version 0.35 of the plugin, that should fix the issue, please let me know if you have further questions
